doc/installing-a-development-environment.md

```diff
@@ -131,6 +135,31 @@ Verify your make installation by running:
 
     make --version
 
+
+### Troubleshooting
+
+#### SSL certificate issue while executing 'gem install bundler' command in Windows
+
+Try copying the certificate from [GlobalSignRootCerficateAuthority] into `C:\Ruby22\lib\ruby\2.2.0\rubygems\ssl_certs` folder
```

Comment:
This only works for this specific version of Ruby and installation path.
I don't think I had to do that when I installed on Windows.
Is there a public link that gives the same advice one can link to?

Reply:
Hi Jan, SSL certificate issue is specific to windows. Also the ruby version and path are inline with the installation steps mentioned in the doc.